omod/src/main/webapp/resources/scripts/services/privilegeService.js

```diff
@@ -0,0 +1,45 @@
+angular.module('privilegeService', ['ngResource'])
```

Comment:
But according to conventions we prefer to have a change in a commit that mentions the ticket number of the ticket it belongs to, there is value in it e.g when tracking which ticket introduced something in the future, it could be a bug or something new.